src/test/java/com/harena/api/service/PatrimoineServiceIT.java

```diff
@@ -24,35 +26,22 @@
 import school.hei.patrimoine.serialisation.Serialiseur;
 
 class PatrimoineServiceIT extends FacadeIT {
-  @MockBean
-  BucketComponent bucketComponent;
+  @MockBean BucketComponent bucketComponent;
   @Autowired PatrimoineService subject;
   private final PatrimoineZetyAu3Juillet2024 patrimoineZetySupplier =
       new PatrimoineZetyAu3Juillet2024();
-  private final Serialiseur<HashMap<String, Patrimoine>> serializer = new Serialiseur<>();
+  private final Serialiseur<Patrimoine> serializer = new Serialiseur<>();
 
   @Test
   void patrimoine_service() throws IOException {
     Path tempFile = Files.createTempFile("test_patrimoine_service", null);
-    HashMap<String, Patrimoine> baseMap = new HashMap<>();
-    baseMap.put("test", patrimoineZetySupplier.get());
-    Files.writeString(tempFile, serializer.serialise(baseMap));
+    Files.writeString(tempFile, serializer.serialise(patrimoineZetySupplier.get()));
     when(bucketComponent.download(any())).thenReturn(tempFile.toFile());
     when(bucketComponent.upload(any(), any())).thenReturn(new FileHash(NONE, "hash"));
 
-    assertFalse(subject.getAllPatrimoine().isEmpty());
-    assertTrue(subject.getPatrimone("test").isPresent());
+    assertFalse(subject.getAllPatrimoine(PageRequest.of(1, 1)).isEmpty());
```

Comment:
Page(1,1) is not representative of reality. People usually use something like Page(4,10)